doc/developers/maintainer.rst.template

```diff
@@ -160,6 +161,24 @@ Reference Steps
         the `tool.towncrier` section in `pyproject.toml`.
     {% endif %}
 
+    - The `scikit-learn/scikit-learn.github.io` needs to be cleaned up so that ideally
+      it stays <5GB in size. Before doing this, create a fork of the existing repo
```

Comment:
Just curious do you know whether the indicative ~5GB acceptable size is for the `.git` or for the total size of the `git clone`. For one commit for scikit-learn `.git` is at `1.6G`, `git clone` is at 5.2G.

MNE-Python which is using a similar strategy (squash commits every release) for one commit `2.3G` `git clone` `7.6G`.

Reply:
It's total size of the repo and its history, so includes `.git` folder. That's the recommendation, but the hard limit is 100GB.